examples/demo/src/layout/Login.js

```diff
@@ -133,13 +139,9 @@ const Login = ({ classes, handleSubmit, isLoading, location, userLogin }) => {
 Login.propTypes = {
     ...propTypes,
     authProvider: PropTypes.func,
-    classes: PropTypes.object,
     previousRoute: PropTypes.string,
-    userLogin: PropTypes.func.isRequired,
 };
 
-const mapStateToProps = state => ({ isLoading: state.admin.loading > 0 });
-
 const enhance = compose(
     translate,
```

Comment:
I need `translate` because redux form use `translate props`. https://github.com/marmelab/react-admin/pull/3345/files#diff-0fa149934be4df88d49bc127277795d2R153

Reply:
Right! I'll be glad when we get rid of that.